### DISCUSSION TITLE
[OBS] Activer / Desactiver Source et filtre

### DISCUSSION THREAD

**Discussion by ghost** [skarab42/marv#113] [Ideas]
Bonjour,
On pourrais faire une activation de filtre et de source via Marv
Comme exemple la cam qui s ecrase via une commande en point de chaine.

**Comment by skarab42** [comment]
Hello, déjà demandé dans #19 et #91 ;) 
Merci de chercher un peu avent de poster en double...

**Reply by ghost** [reply]
ok je m en excuse , je n avais pas bien fouiller dans les idees

**Reply by skarab42** [reply]
Pas de soucis ;) Merci a toi de prendre le temps de proposer des choses^^
